lib/auth/tls_test.go

```diff
@@ -1603,6 +1604,31 @@ func (s *TLSSuite) TestAccessRequest(c *check.C) {
 	userClient, err := s.server.NewClient(testUser)
 	c.Assert(err, check.IsNil)
 
+	// Verify that user has correct requestable roles
+	caps, err := userClient.GetAccessCapabilities(context.TODO(), services.AccessCapabilitiesRequest{
+		RequestableRoles: true,
+	})
+	c.Assert(err, check.IsNil)
+	c.Assert(caps.RequestableRoles, check.HasLen, 1)
```

Comment:
It is usually better to be more explicit about the expected value state:
```go
c.Assert(caps.RequestableRoles, check.DeepEquals, []string{role})
```
or even:
```go
c.Assert(caps, check.DeepEquals, services.AccessCapabilities{
	RequestableRoles: []string{role},
})
```